packages/vm/src/dynamic_link.rs

```diff
@@ -234,6 +236,56 @@ where
     Ok(write_to_contract(env, value)?.into())
 }
 
+pub fn native_validate_dynamic_link_interface<A: BackendApi, S: Storage, Q: Querier>(
+    env: &Environment<A, S, Q>,
+    address: u32,
+    interface: u32,
+) -> Result<u32, RuntimeError>
+where
+    A: BackendApi + 'static,
+    S: Storage + 'static,
+    Q: Querier + 'static,
+{
+    let contract_addr_raw = read_region(&env.memory(), address, 64)?;
+    let contract_addr: Addr = from_slice(&contract_addr_raw)
+        .map_err(|_| RuntimeError::new("Invalid contract address to detect interface"))?;
```

Comment:
It's a matter of word, Are "detect" and "validate" used interchangeably?.
(All other places are validate, but only here is detect.)

Reply:
nice found. thank you.